### PR TITLE
Add MacBookPro5,3 to TESTED.md

### PR DESCRIPTION
**Pull request by kommtzeitkonrad**

### CODE REVIEW

File: docs/TESTED.md [hunk 1]
```diff
@@ -28,17 +28,17 @@ To aid users in troubleshooting, we've compiled a list of users who've reported
...
 | SMBIOS | Tested | Tester | Comment |
 | :--- | :--- | :--- | :--- |
-| MacBookPro4,1 | <span style="color:#30BCD5"> YES </span> | cboukouv | - Stock model <br/>- Patcher version 0.0.19  |
+| MacBookPro4,1 | <span style="color:#30BCD5"> YES </span> | cboukouv | - Stock model <br/>- Patcher version 0.0.19 |
 | MacBookPro5,1 | ^^ | GURU_Jasho#5736 | - Stock model <br/>- Patcher version 0.0.21 |
-| MacBookPro5,2 |  ^^ | hvds | ^^ |
-| MacBookPro5,3 | No reports | N/A | N/A |
-| MacBookPro5,4 | ^^ | ^^ | ^^ |
+| MacBookPro5,2 | ^^ | hvds | ^^ |
+| MacBookPro5,3 | <span style="color:#30BCD5"> YES </span> | kommtzeitkonrad | - Stock model <br/>- Patcher version 0.0.22 |
```
Comment (khronokernel) [inline docs/TESTED.md:40]: `<span style="color:#30BCD5"> YES </span>` should be replaced with `^^` to have proper formatting in the with the site

**Review by khronokernel — CHANGES_REQUESTED** [review]

**Comment by khronokernel** [comment]
PR looks good to me, have a small change requested but otherwise will merge. Thanks for the PR!

**Comment by kommtzeitkonrad** [comment]
Thank you for how amazing this patcher has become!
I must say that I would really like to get my hands on the early developer preview of the GPU acceleration patches to test it on the MBP5,3 and my good old HD3000-powered Hackintosh. :-)

**Comment by khronokernel** [comment]
Awesome thanks for the PR! Are you available on discord? Currently all our testing is on the Unsupported Mac discord, as it's easier for us to consolidate testing. If so, I can add you to the beta channel. Just need to know your discord username

**Comment by kommtzeitkonrad** [comment]
kommtzeitkonrad#0254 :-)

**Pull request merged**